containers/runner/launch

```diff
@@ -41,12 +54,14 @@ Options:
                                        rhinstaller/kickstart-tests workflow artifacts.)
  --defaults DEFAULTS_SH_FILE          Path to file with overrides to scripts/defaults.sh
  --run-args ARGUMENTS                 Extra $CRUN options/arguments (space separated)
+ --recommended-jobs                   Get the number of jobs (--jobs option) run in parallel
+                                      by default, based on number of CPUs and RAM detected
```

Comment:
Could you please improve the message to be obvious what this option is doing?

Something like:
``Print number of jobs (see --jobs option) used by default and exit. These jobs are calculated from number of CPU and size of RAM.``

Reply:
Updated

Reply:
@jkonecny12 - is it ok now?